<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>formAyarlar</class>
 <widget class="QDialog" name="formAyarlar" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Ayarlar</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="simgeler.qrc" >:/new/prefix1/icons/configure.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Genel</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="3" column="1" >
          <widget class="QLineEdit" name="lineUrl" />
         </item>
         <item row="2" column="1" >
          <widget class="QLineEdit" name="lineTelefon" />
         </item>
         <item row="1" column="1" >
          <widget class="QLineEdit" name="lineAdres" />
         </item>
         <item row="3" column="0" >
          <widget class="QLabel" name="label_8" >
           <property name="text" >
            <string>İ&amp;nternet Adresi</string>
           </property>
           <property name="buddy" >
            <cstring>lineUrl</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="1" >
          <widget class="QLineEdit" name="linePosta" />
         </item>
         <item row="0" column="0" >
          <widget class="QLabel" name="label_5" >
           <property name="text" >
            <string>Ş&amp;irket / Kullanıcı Adı:</string>
           </property>
           <property name="buddy" >
            <cstring>lineSirket</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_6" >
           <property name="text" >
            <string>&amp;Adres</string>
           </property>
           <property name="buddy" >
            <cstring>lineAdres</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="0" >
          <widget class="QLabel" name="label_9" >
           <property name="text" >
            <string>&amp;E-Posta</string>
           </property>
           <property name="buddy" >
            <cstring>linePosta</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QLineEdit" name="lineSirket" />
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="label_7" >
           <property name="text" >
            <string>&amp;Telefon</string>
           </property>
           <property name="buddy" >
            <cstring>lineTelefon</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3" >
      <attribute name="title" >
       <string>Yazıcı</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="label_10" >
           <property name="text" >
            <string>Sayfa Boyutu : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboPrinter" >
           <item>
            <property name="text" >
             <string>A4</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>A5</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>A3</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>E-Posta</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="2" column="0" >
          <widget class="QLabel" name="label_3" >
           <property name="text" >
            <string>&amp;Kullanıcı Adı</string>
           </property>
           <property name="buddy" >
            <cstring>lineKullanici</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QLineEdit" name="lineSmtp" />
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>&amp;Port (25)</string>
           </property>
           <property name="buddy" >
            <cstring>spinPort</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QLineEdit" name="lineKullanici" />
         </item>
         <item row="3" column="1" >
          <widget class="QLineEdit" name="lineParola" >
           <property name="echoMode" >
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QSpinBox" name="spinPort" >
           <property name="maximum" >
            <number>65535</number>
           </property>
           <property name="minimum" >
            <number>1</number>
           </property>
           <property name="value" >
            <number>25</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0" >
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>&amp;SMTP Adresi</string>
           </property>
           <property name="buddy" >
            <cstring>lineSmtp</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="0" >
          <widget class="QLabel" name="label_4" >
           <property name="text" >
            <string>P&amp;arola</string>
           </property>
           <property name="buddy" >
            <cstring>lineParola</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>Kaydet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>İptal</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>lineSirket</tabstop>
  <tabstop>lineAdres</tabstop>
  <tabstop>lineTelefon</tabstop>
  <tabstop>lineUrl</tabstop>
  <tabstop>linePosta</tabstop>
  <tabstop>comboPrinter</tabstop>
  <tabstop>lineSmtp</tabstop>
  <tabstop>spinPort</tabstop>
  <tabstop>lineKullanici</tabstop>
  <tabstop>lineParola</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources>
  <include location="simgeler.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>formAyarlar</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>389</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
